docs/docs/noir/standard_library/meta/fmtstr.md

```diff
@@ -0,0 +1,13 @@
+---
+title: fmtstr
+---
+
+`std::meta::format_string` contains comptime methods on the `fmtstr` type for format strings.
```

Comment:
Can you move this file to be a general docs for `fmtstr`? Users shouldn't know/care that these methods are in `std::meta::format_string` since they just want to  see all the methods on `fmtstr` in one place.